src/components/DeviceDetails/__tests__/DeviceDetails.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { omit } from "lodash";
+import { cleanup } from "@testing-library/react";
+import { render } from "@deskpro/app-testing-utils";
+import { DeviceDetails } from "../DeviceDetails";
+import { mockDevices } from "../../../../testing";
+import type { Props } from "../DeviceDetails";
+
+const mock = {
+  site: omit(mockDevices.data.authorizedSites.sites[0], ["assetResources"]),
+  ...mockDevices.data.authorizedSites.sites[0].assetResources.items[1],
+};
+
+const renderDeviceDetails = (props?: Partial<Props>) => render((
+  <DeviceDetails
+    device={props?.device || mock as never}
```

Comment:
Why do we not pass in the correct types here. It should be possible to do testing whilst being typesafe.

Reply:
added types

Reply:
already fixed